otx/algorithms/detection/configs/instance_segmentation/efficientnetb2b_maskrcnn/data_pipeline.py

```diff
@@ -0,0 +1,80 @@
+"""Data Pipeline of EfficientNetB2B model for Instance-Seg Task."""
+
+# Copyright (C) 2022 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+
+# pylint: disable=invalid-name
+
+dataset_type = "CocoDataset"
+img_size = (1024, 1024)
+
+img_norm_cfg = dict(mean=(103.53, 116.28, 123.675), std=(1.0, 1.0, 1.0), to_rgb=False)
```

Comment:
Is to_rgb=False on purpose? In detection all models pipeline's to_rgb is True

Reply:
This is the same setting as the latest develop.

Reply:
This is the value used in all experiments so far, so it is better to leave it as it is.
This seems to require experimentation for comparing. Let's do it later.

Reply:
Yeah sure, how about adding TODO flag?

Reply:
Added TODO flag.